internal/hcsoci/create.go

```diff
@@ -109,8 +109,9 @@ func CreateContainer(createOptions *CreateOptions) (_ *hcs.System, _ *Resources,
 	if coi.Spec.Windows != nil &&
 		coi.Spec.Windows.Network != nil &&
 		schemaversion.IsV21(coi.actualSchemaVersion) {
-
-		if coi.NetworkNamespace != "" {
+		if coi.Spec.Windows.Network.NetworkNamespace != "" {
```

Comment:
We want to do this via my above comment. We should only use the `NetworkNamespace` resource value from the create options.

Reply:
Looking into runhcs to understand what is currently done for networking